specification/ekg/resource-manager/Microsoft.EKG/preview/2018-12-03/ekgSwagger.json

```diff
@@ -0,0 +1,586 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Specifies the location of the resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Specifies the type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the SKU of the resource."
+        },
+        "kind": {
+          "$ref": "#/definitions/Kind",
+          "description": "Required. Gets or sets the Kind of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Entity Tag"
+        }
+      }
+    },
+    "Sku": {
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/SkuName",
+          "description": "The sku name"
+        },
+        "tier": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the sku tier. This is based on the SKU name.",
+          "enum": [
+            "Free",
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "The SKU of the cognitive services account."
+    },
+    "SkuName": {
+      "type": "string",
+      "description": "The name of SKU.",
+      "enum": [
+        "F0",
+        "S1"
+      ],
+      "x-ms-enum": {
+        "name": "SkuName",
+        "modelAsString": true
+      }
+    },
+    "Kind": {
+      "type": "string",
+      "description": "Indicates the type of EKG service",
+      "enum": [
+        "sdk",
+        "designer",
+        "EKG",
+        "function"
+      ],
+      "x-ms-enum": {
+        "name": "Kind",
+        "modelAsString": true
+      }
+    },
+    "EKG": {
+      "type": "object",
+      "description": "EKG resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/EKGProperties",
+          "description": "The set of properties specific to EKG resource"
+        }
+      }
+    },
+    "EKGProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The Name of the EKG"
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the EKG"
+        },
+        "iconUrl": {
+          "type": "string",
+          "description": "The Icon Url of the EKG"
+        },
+        "endpoint": {
```

Comment:
For endpoint, endpointVersion, msAppId, developer*, luis* - I understand that these are not used. Please remove